calico_kubernetes/logutils.py

```diff
@@ -0,0 +1,18 @@
+#!/bin/python
+import logging
+import os
+
+LOG_DIR = '/var/log/calico/kubernetes'
+
+def configure_logger(logger, log_dir=None):
+    if log_dir is None:
+        log_dir = LOG_DIR
+
+    logging._srcfile = None
```

Comment:
Its not really necessary but by setting _srcfile to None, the logger does not collect information on where the calls are coming from